src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -236,7 +236,28 @@ object ModelJsonProtocol extends WorkspaceJsonSupport {
   implicit val impLibrarySearchResponse = jsonFormat4(LibrarySearchResponse)
   implicit val impLibraryBulkIndexResponse = jsonFormat3(LibraryBulkIndexResponse)
 
-  implicit val impDuosConsent = jsonFormat10(Consent)
+  implicit object impDuosDataUse extends RootJsonFormat[DuosDataUse] {
+    override def write(ddu: DuosDataUse): JsValue = {
+      val existingProps: Seq[(String, JsValue)] = ddu.getClass.getDeclaredFields map { f =>
```

Comment:
In this case, it's not handled at all. Any exception would propagate up to the caller, so it should result in a 500 error when orch tries to return it in json format.

Reply:
Wouldn't it be better to log something useful for debug purposes and/or return something friendly to the user? I'm not familiar with the use case so I can't tell.

Reply:
Yes, it would be nice to handle this better. The current `jsonFormatX` all throw `deserializationError` exceptions along with `serializationError` exceptions - I'll add that.